cloud/aws/alb/variables.tf

```diff
@@ -11,9 +11,9 @@ variable "notifications" {
   description = "Default notification recipients list per severity"
   type = object({
     critical = list(string)
-    major    = list(string)
```

Comment:
The order of severities doesn't respect the SignalFX order.
It's better to have :
- critical
- major
- minor
- warning
- info

This change appears into all modules.